src/test/java/dev/openfeature/sdk/testutils/TestFlagsUtils.java

```diff
@@ -0,0 +1,73 @@
+package dev.openfeature.sdk.testutils;
+
+import com.google.common.collect.ImmutableMap;
+import dev.openfeature.sdk.Value;
+import dev.openfeature.sdk.providers.memory.Flag;
+import lombok.experimental.UtilityClass;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static dev.openfeature.sdk.MutableStructure.mapToStructure;
+
+/**
+ * Test flags utils.
+ */
+@UtilityClass
+public class TestFlagsUtils {
+
+    /**
+     * Building flags for testing purposes.
+     * @return map of flags
+     */
+    public static Map<String, Flag<?>> buildFlags() {
```

Comment:
I think this API looks pretty solid. It's what I expected. Interested in @justinabrahms @thisthat 's takes.